components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksFeature.kt

```diff
@@ -52,7 +54,8 @@ class AppLinksFeature(
     private val launchInApp: () -> Boolean = { false },
     private val useCases: AppLinksUseCases = AppLinksUseCases(context, launchInApp),
     private val failedToLaunchAction: () -> Unit = {},
-    private val loadUrlUseCase: SessionUseCases.DefaultLoadUrlUseCase? = null
+    private val loadUrlUseCase: SessionUseCases.DefaultLoadUrlUseCase? = null,
```

Comment:
Why does this have a `null` default? would there ever be a case where the scheme is supported but we don't have this use case?

Reply:
There could be a case that the user of `AppLinksFeature` decides not to use the LoadUrlUseCase.  In that case they can pass in a null and we won't try to load URLs through `AppLinksFeature`.